docs/source/deploy/sandbox.rst

```diff
@@ -0,0 +1,59 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _deploy-sandbox:
+
+Deploying to a generic DAML ledger
+==================================
+
+DAML ledgers expose a unified administration API. This means that deploying to a DAML ledger is no
+different from deploying to your local sandbox.
+
+To deploy to a DAML ledger, run the following command from within your DAML project:
+
+.. code-block:: none
+
+   $ daml deploy --host=<HOST> --port=<PORT> --access-token-file=<TOKEN-FILE>
+
+where ``<HOST>`` and ``<PORT>`` is the hostname and port your ledger is listening on, which defaults
+to port ``6564``. The ``<TOKEN-FILE>`` is needed if your sandbox runs with :ref:`authentication
+<authentication>` and needs to contain a JWT token with an ``admin`` claim. If your sandbox is not
+setup to use any authentication it can be omitted.
+
+Instead of passing ``--host`` and ``--port`` flags to the command above, you can add the following
+section to the project's ``daml.yaml`` file:
+
+.. code-block:: yaml
+
+   ledger:
+       host: <HOSTNAME>
+       port: <PORT>
+
+The ``daml deploy`` command will
+
+#. upload the project's compiled DAR file to the ledger. This will make the DAML templates defined
+   in the current project available to the API users of the sandbox.
+
+#. allocate the parties specified in the project's ``daml.yaml`` on the ledger if they are missing.
```

Comment:
It might be helpful to spell out that `dam deploy` is `daml build` + `daml ledger upload-dar` + `daml ledger allocate-parties`.

Reply:
I'd like to keep this as is, otherwise, we just repeat what has just been said in the text and there is a full listing of daml ledger commands in the next paragraph.